### PR TITLE
[OLNM-12][Feat] (i/o)Rent label

### PR DESCRIPTION
**Pull request by dehatanes**
## CONTEXT
In order to easily identify (i|o)Rent houses, it must be shown with a “iRent”/”oRent” tag in the page of that house.

JIRA [OLNM-12](https://quintoandar.atlassian.net/jira/software/projects/OLNM/boards/235?selectedIssue=OLNM-12)(**(i|o)Rent Properties Tag**)

## LAYOUT
Screenshot

### CODE REVIEW

File: qa-imovel-infobox/qa-imovel-infobox.js [hunk 1]
```diff
@@ -35,20 +35,29 @@ Polymer(
...
+				const ioRentSpecialCondition = self.imovel.specialConditions.filter( function(specialCondition) {
+					return specialCondition.specialConditionType === 'IRent'
+						|| specialCondition.specialConditionType === 'ORent';
+				} )[0];
+
 				self.originals = originalsSpecialCondition ?
 					originalsSpecialCondition.specialConditionType.replace('Originals', 'Originals ') : null ;
+
+				self.ioRent = ioRentSpecialCondition ?
+					ioRentSpecialCondition.specialConditionType.replace('Rent', '\'Rent ') : null ;
```
Comment (fvcaputo) [inline qa-imovel-infobox/qa-imovel-infobox.js]: This makes the tag appears as `I'Rent` and `O'Rent`, is this the expected behavior? Shouldn't it be `iRent` and `oRent`?

[the remark this answers is not in the document]
Comment (baiyjmatheus) [reply qa-imovel-infobox/qa-imovel-infobox.js]: return string.substring(0,1).toLowerCase() + string.substring(1);

[the remark this answers is not in the document]
Comment (dehatanes) [reply qa-imovel-infobox/qa-imovel-infobox.js]: done! :D Thanks

Comment (dehatanes) [reply qa-imovel-infobox/qa-imovel-infobox.js]: done! Thanks :D

**Pull request merged**